doc/docs_es.md

```diff
@@ -0,0 +1,899 @@
+y al final habrás aprendido prácticamente todo el lenguaje.
+
+El lenguaje promueve la escritura de código simple y claro con mínima abstracción.
+
+A pesar de ser simple, V le da mucho poder al desarrollador.
+Todo lo que puedas hacer en otros lenguajes, lo puedes hacer en V.
+
+## Instalación desde el código fuente
+
+La forma principal de obtener la versión de V más reciente y mejorada es
+__instalarlo desde el código fuente__.
+Es __fácil__ y, por lo general, toma __sólo unos segundos__.
+
+### Linux, macOS, FreeBSD, etc:
+
+Solo necesitas `git`, un compilador de C como `gcc` o `clang`, y `make`:
+
+```bash
+git clone https://github.com/vlang/v && cd v && make
+```
+
+### Windows
+
+Solo necesitas `git`, y un compilador de C como `gcc` o `msvc`:
+
+```bash
+git clone https://github.com/vlang/v
+cd v
+make
+```
+
+### Android
+
+También es posible ejecutar aplicaciones gráficas hechas con V en Android
+a través de [vab](https://github.com/vlang/vab).
+
+Las dependencias que necesita V en Android son: **V** (obviamente), **JavaSDK** >= 8
+y Android **SDK + NDK**.
+
+  1. Instale las dependencias (mirar [vab](https://github.com/vlang/vab))
+  2. Conecte tu dispositivo Android a la PC
+  3. Ejecute:
+  ```bash
+  git clone https://github.com/vlang/vab && cd vab && v vab.v
+  ./vab --device auto run /path/to/v/examples/sokol/particles
+  ```
+
+Para más detalles y solución de problemas, por favor visite el repositorio en Github de
+[vab](https://github.com/vlang/vab).
+
+## Tabla de contenidos
+
+<table>
+    <tr><td width=33% valign=top>
+
+* [Hola Mundo](#hola-mundo)
+* [Comentarios](#comentarios)
+* [Funciones](#funciones)
+    * [Retornando múltiples valores](#retornando-m%C3%BAltiples-valores)
+    * [Número variable de argumentos](#n%C3%BAmero-variable-de-argumentos)
+* [Visibilidad de un símbolo](#visibilidad-de-un-s%C3%ADmbolo)
+* [Variables](#variables)
+    * [Variables mutables](#variables-mutables)
+    * [Inicialización vs asignamiento](#inicializaci%C3%B3n-vs-asignamiento)
+    * [Errores de declaración](#errores-de-declaraci%C3%B3n)
+* [Tipos](#tipos)
+    * [Tipos primitivos](#tipos-primitivos)
+    * [Strings (Cadenas)](#strings-cadenas)
+    * [Números](#n%C3%BAmeros)
+    * [Matrices (Arrays)](#matrices-arrays)
+    * [Matrices de tamaño fijo](#matrices-de-tamaño-fijo)
+    * [Mapas](#mapas)
+* [Importación de módulos](#importaci%C3%B3n-de-m%C3%B3dulos)
+</td></tr>
+</table>
+
+## Hola Mundo
+
+```v
+fn main() {
+	println('hola mundo')
+}
+```
+
+Guarda este fragmento de código en un archivo llamado "hello.v". Ahora ejecuta: `v run hello.v`.
+
+> Esto se hace asumiendo que ha enlazado V usando `v symlink`, como se describe
+[aquí (en inglés)](https://github.com/vlang/v/blob/master/README.md#symlinking).
+Si aún no lo has hecho, debes escribir la ruta al ejecutable de V manualmente.
+
+¡Felicitaciones, acaba de escribir y ejecutar su primer programa hecho en V!
```

Comment:
Minor nitpick, but I feel like the docs should either be written entirely in third person (`usted` - more formal), or entirely in second person (`tú` - informal). Switching between the two is a little weird :)

Reply:
Yes, I try to write the document in the second person. This detail had escaped me, thank you :).